x-pack/test_serverless/api_integration/test_suites/common/alerting/helpers/alerting_wait_for_helpers.ts

```diff
@@ -366,13 +368,17 @@ export async function waitForExecutionEventLog({
 }
 
 export async function waitForNumRuleRuns({
-  supertest,
+  supertestWithoutAuth,
+  roleAuthc,
+  internalReqHeader,
   numOfRuns,
   ruleId,
   esClient,
   testStart,
 }: {
-  supertest: SuperTestAgent;
+  supertestWithoutAuth: SupertestWithoutAuthType;
+  roleAuthc: RoleCredentials;
+  internalReqHeader: InternalRequestHeader;
```

Comment:
Similar to my comment in `x-pack/test_serverless/api_integration/services/alerting_api.ts` - but here we are not in the context of a service.
We could leave this as is, but the proper solution would be to convert this helper file into an actual service (or include it in the alerting_api service). Could also leave it as a follow-up for the test owners.

Reply:
I'd like to follow this up, right after this pr.